chef/cookbooks/crowbar/recipes/crowbar-upgrade.rb

```diff
@@ -104,7 +104,7 @@
       for type in clone ms primitive; do
         for resource in $(crm configure show | grep ^$type | cut -d " " -f2);
         do
-          crm resource stop $resource
+          crm --force --wait resource stop $resource
```

Comment:
It's true that `--force` was likely not needed so far. But it's probably better to be sure if some user interaction would be required...